data/net/minecraft/client/server/IntegratedServer.mapping

```diff
@@ -1,4 +1,16 @@
 CLASS net/minecraft/client/server/IntegratedServer
+	METHOD <init> (Ljava/lang/Thread;Lnet/minecraft/client/Minecraft;Lnet/minecraft/core/RegistryAccess$RegistryHolder;Lnet/minecraft/world/level/storage/LevelStorageSource$LevelStorageAccess;Lnet/minecraft/server/packs/repository/PackRepository;Lnet/minecraft/server/ServerResources;Lnet/minecraft/world/level/storage/WorldData;Lcom/mojang/authlib/minecraft/MinecraftSessionService;Lcom/mojang/authlib/GameProfileRepository;Lnet/minecraft/server/players/GameProfileCache;Lnet/minecraft/server/level/progress/ChunkProgressListenerFactory;)V
+		ARG 1 thread
+		ARG 2 mc
+		ARG 3 registryHolder
+		ARG 4 levelStorageAccess
+		ARG 5 packRepository
+		ARG 6 serverResources
+		ARG 7 worldData
+		ARG 8 session
+		ARG 9 gameProfileRepository
+		ARG 10 gameProfileCache
+		ARG 11 chunkProgressionListenerfactory
```

Comment:
Parameter names should follow the name of the field they're assigned to. (These are from `MinecraftServer.<init>`.)
```suggestion
		ARG 1 serverThread
		ARG 2 minecraft
		ARG 3 registryHolder
		ARG 4 storageSource
		ARG 5 packRepository
		ARG 6 resources
		ARG 7 worldData
		ARG 8 sessionService
		ARG 9 profileRepository
		ARG 10 profileCache
		ARG 11 progressListenerfactory
```